src/datasources/products/ProductsDataSource.ts

```diff
@@ -65,14 +65,18 @@ export class ProductsDataSource extends DataSourceBase<ProductQuery> {
     returnCount = false
   ): Promise<QueryProductResponse> {
     try {
-      const response = await this.post<QueryProductResponse>(this.queryProductsUrl, {
-        filter,
-        orderBy,
-        descending,
-        projection,
-        take,
-        returnCount
-      });
+      const response = await this.post<QueryProductResponse>(
+        this.queryProductsUrl,
+        {
+          filter,
+          orderBy,
+          descending,
+          projection,
+          take,
+          returnCount
+        },
+        false
```

Comment:
Consider using an options object instead of adding a flag. Someone reviewing this code in the future will not have a clear understanding of what this random `false` flag is doing at first glance. Can express this pretty well with parameter defaults and destructuring assignment

```
post<T>(url: string, body: Record<string, any>, {
    showErrorAlert = true,
} = { 
    showErrorAlert: true
})
```

```
this.post(..., {showErrorAlert: false})
```